### PR TITLE
Nikita/Add funds redirect button in MT5 manage funds not redirecting to cashier page in one click.

### PR DESCRIPTION
**Pull request by nikita-deriv**
Link "add funds" opens on first click

### CODE REVIEW

File: src/javascript/index.js [hunk 1]
```diff
@@ -24,3 +24,4 @@ $(window).on('pageshow', (e) => { // Safari doesn't fire load event when using b
         BinaryLoader.init();
     }
 });
+
```
Comment (akmal-binary) [inline src/javascript/index.js:27]: There is an extra empty line in the end, can we remove it?

File: src/javascript/app/common/guide.js [hunk 2]
```diff
@@ -43,12 +44,8 @@ const Guide = (() => {
             $(opt.guideBtnID).remove();
             return;
         }
-
         makeButton();
     };
-    /*
```
Comment (akmal-binary) [inline src/javascript/app/common/guide.js:49]: Any reason we are removing this comment? Seems useful and consistent with other functions. Maybe we can discard changes on this file if it's not intentional.

File: src/javascript/index.js (hunk 1 above)
Comment (nikita-deriv) [reply src/javascript/index.js]: Looks like was added unnecessary changes, removed all, thanks.

File: src/javascript/app/common/guide.js (hunk 2 above)
Comment (nikita-deriv) [reply src/javascript/app/common/guide.js]: Returned.

**Review by akmal-binary — APPROVED** [review]

**Pull request merged**